src/libzippp.h

```diff
@@ -356,6 +356,7 @@ namespace libzippp {
          **/
         bool isEntryCompressionEnabled(const ZipEntry& entry) const;
         bool setEntryCompressionEnabled(const ZipEntry& entry, bool value) const;
```

Comment:
This line has to be removed, the method doesn't exist anymore.

Reply:
done